governance/ApiDeprecation.md

```diff
@@ -0,0 +1,121 @@
+# API Deprecation
+
+## Notice
+
+> Wondering if this applies to you? This process only applies to __public__ APIs. If you're sure consumers of your API will not be impacted, making an immediate change is okay.
+
+## Info
+
+> Unsure about what type of deprecation you're facing? Please see this page for more information - [Deprecation Meta](DeprecationMeta.md)
+
+## Overview
+
+Examples of a API deprecation:
+
+- Renaming a function
+- Renaming a class/EBus
+- Changing a function signature
+- Removing a function
+- Removing a class/EBus
+- Changing the address type of an EBus
+- Changing the preconditions of a function
+  - e.g. Function no longer handles `nullptr`s or parameter values must now fall within a specified range (e.g. between `0.0` and `1.0`).
+
+## Process
+
+### Golden Rule
+
+> Do not make any breaking changes to an API when first deprecating it (this includes adding the `AZ_DEPRECATED` macro). The code must work exactly as before.
+
+### Steps (Guide)
+
+1. Identify the code to be deprecated (something resembling one of the API deprecation examples outlined in the Overview).
+    1. _Note: It's fine to group several related changes into a single deprecation_.
+1. Create a new GitHub issue using the Deprecation issue template (this will come with all the required labels/milestones).
```

Comment:
Do we need a new deprecation issue type? Why add yet another issue type. Its not clear for the value of this over the friction of standard issue creation.

Small deprecations may have an issue, but many are going to be tied to RFCs (ie RFC to replace system A)

Reply:
We do already have a label that is `kind/deprecation` so we should be good there (at the very least for API deprecation)